packages/common/src/index.ts

```diff
@@ -154,6 +155,21 @@ export const validateEmail = (email: string): boolean => {
     return re.test(String(email).toLowerCase());
 };
 
+export function validatePassword(password: string): PasswordValidationResult {
+    const minLength = 8;
+    const hasLetter = /[a-zA-Z]/.test(password);
+    const hasNumberOrSymbol = /[\d\W]/.test(password);
```

Comment:
This regex should capture all symbols, however it does not work with `_` :thinking: do we consider that a symbol ? 
![image](https://github.com/lightdash/lightdash/assets/1983672/ca819f14-fe4d-4270-b70f-6882093fda97)

Reply:
Should we include it? Not sure whether we include this as a symbol or not

Reply:
Yes, I think we should include `_` it should be ` /[\d\W_]/` but let's create an unit test to make sure